src/ui/public/indices/get_indices.js

```diff
@@ -1,6 +1,10 @@
 import { pluck, reduce, size } from 'lodash';
 
-const getIndicesFromResponse = json => {
+const getIndexNamesFromAliasesResponse = json => {
+  if (json.status === 404) {
```

Comment:
I'd be OK removing this check if we were to scope both helpers to the caller, e.g. by moving them to line 27. Want me to do that? It's just that since they're outside, they look like they're more generalized helper functions, which tend to be a little more bullet proof.